### PR TITLE
Examples: require to require_once; PHP 5 to 7

### PR DESCRIPTION
**Pull request by lsthompson**
Seems happy on PHP7, though we've implemented the class for PMG not PVE. Updated examples from require to require_once.

### CODE REVIEW

**Comment by lsthompson** [comment]
Hey, is this okay to merge?

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Example - Create a new OpenVZ Container on the first host in the cluster.
         // print_r($new_container_settings);
         print("---------------------------\n");
 
-        print_r($pve2->post("/nodes/".$first_node."/openvz", $new_container_settings));
+        print_r($pve2->post("/nodes/".$first_node."/openvz",$new_container_settings));
```
Comment (danhunsaker) [inline README.md:60]: Should probably leave the space here.

File: README.md [hunk 2]
```diff
@@ -84,15 +84,15 @@ Example - Modify DNS settings on an existing container on the first host.
         $container_settings['nameserver'] = "4.2.2.2";
 
         # NOTE - replace XXXX with container ID.
-        var_dump($pve2->put("/nodes/".$first_node."/openvz/XXXX/config", $container_settings));
+        var_dump($pve2->put("/nodes/".$first_node."/openvz/XXXX/config",$container_settings));
```
Comment (danhunsaker) [inline README.md:87]: Same as above, leave the space.

File: README.md [hunk 3]
```diff
@@ -7,13 +7,13 @@ API spec available at https://pve.proxmox.com/pve-docs/api-viewer/index.html
 
 ## Requirements: ##
 
-PHP 5 with cURL (including SSL) support.
+PHP 7 with cURL (including SSL/TLS) support.
```
Comment (danhunsaker) [inline README.md:10]: Does this no longer work on 5.x? I'd feel better saying "5 or higher" if it isn't actually broken there. I know 5 is massively outdated, but I'm sure it's still in the wild somewhere.

**Review by danhunsaker — CHANGES_REQUESTED** [review]

**Review by lsthompson — COMMENTED** [review]
Good pick-ups @danhunsaker. Now tweaked - thanks.

**Review by danhunsaker — APPROVED** [review]

**Pull request merged**